source/inventorymenu.cpp

```diff
@@ -128,6 +132,29 @@ bool InventoryMenu::modeChange(MenuButton * button, void * param)
     return true;
 }
 
+bool InventoryMenu::getAllCharsPixls(MenuButton * button, void * param)
+{
+    (void) button;
+    InventoryMenu * instance = reinterpret_cast<InventoryMenu *>(param);
+    spm::mario_pouch::MarioPouchWork * pp = spm::mario_pouch::pouchGetPtr();
+    ModeDef chars = modes[3];
```

Comment:
Slightly more efficient to take const references to the mode defs, this would create copies of the whole def on the stack